src/helpers.js

```diff
@@ -855,3 +855,51 @@ export function getNextAvailableTarget(existingTargets, busType) {
         i++;
     }
 }
+
+/**
+ * Parses output of "udevadm info --path [devicePath]" into a node device object
+ * Inspired by parseUdevDB() from cockpit's pkg/lib/machine-info.js
+ *
+ * @param {string} text output of "udevadm" commdn
+ * @returns {object}
+ */
+export function parseUdevDB(text) {
```

Comment:
This is not used anywhere. The function now exists in src/libvirtApi/nodeDevice.js